frontend/src/App.css

```diff
@@ -0,0 +1,6 @@
+.cardImg
+{
+    padding: 1rem;
+    min-height: '200px';
+    min-width: '200px';
+}
```

Comment:
I should have mentioned to put this inside of a new file named FilledCard.css. You will also have to create a new folder under src named styles and put the newly created FilledCard.css inside of that folder.